Add back failing test to run it on CI

iangmaia opened the pull request:
Add back test removed on #15214 to make sure the CI test report is working properly

Internal reference: p1740053759413429-slack-CC7L49W13

iangmaia committed:
Add back failing test to run it on CI

dangermattic commented:
<!--
  0 Errors
  0 Warnings
  1 Message: This PR is still a Draft: some...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

iangmaia committed:
Force failure

AliSoftware committed:
Update to latest fastlane

Which contains the fix for trainer to support SwiftTesting and correctly report test failures on those
